recipe/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "wasmtime" %}
+{% set version = "11.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/bytecodealliance/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 681d03d9ebf63697edf8b6c7df460aca6ecc0058106b07f8ac049a0b2a07308b
+
+  - url: https://github.com/WebAssembly/WASI/archive/0ba0c5e2e37625ca5a6d3e4255a998dfaa3efc52.tar.gz
+    sha256: 1a0dce1333fa5d588a09d81acd63a8eb1e13296768f24cb348ceffaf5e6f51fd
+    folder: crates/wasi-common/WASI
+
+  - url: https://github.com/WebAssembly/wasi-crypto/archive/67d9821f2763f4626a97c70c411ccb92a2047712.tar.gz
+    sha256: 49ebe4479ffefe191208b9bc7ab9588fc9ce896ef43221d0d241789a71224bd1
+    folder: crates/wasi-crypto/spec
+
+  - url: https://github.com/WebAssembly/wasi-nn/archive/8adc5b9b3bb8f885d44f55b464718e24af892c94.tar.gz
+    sha256: 65748477f6d834c2d41de6bbe773254c395e4ddeae3461f84997c58dbf6276be
+    folder: crates/wasi-nn/spec
+
+  - url: https://github.com/WebAssembly/wasi-http/archive/1c95bc21dbd193b046e4232d063a82c8b5ba7994.tar.gz
+    sha256: 8d0a31a4265f610627a86aa7766d4f511b23a19f9c06d6c9febbf29f02cb4f64
+    folder: crates/wasi-http/wasi-http
+
+build:
+  number: 0
+  skip: True # [linux and (s390x or ppc64le)]
```

Comment:
Perhaps add a comment why these are skipped?

Reply:
we can try to build it, but those are not supported by wasmtime-py and snowflake does not need those
see my comment on wasmtime-py
https://github.com/AnacondaRecipes/wasmtime-py-feedstock/pull/1#discussion_r1298218854